### PR TITLE
Fix datetime stored in database

### PR DESCRIPTION
**Pull request by julien4215**
Currently Titan gives to Hyperion local datetime that we are storing in the database as timestamp with timezone. The problem is that they are never converted to utc so we have wrong values in the database. 

We should check if a timezone is set in the `postgresql.conf` and set it to Etc/UTC if it is not the case.

Tracking issue: https://github.com/aeecleclair/Titan/issues/114

### CODE REVIEW

**Review by guillaumetavernier — COMMENTED** [review]
We can still find reference to `TIMEZONE` in docker compose files and in config.py. Did you check those ?

**Comment by julien4215** [comment]
> We can still find reference to `TIMEZONE` in docker compose files and in config.py. Did you check those ?

Yes I will add again the timezone parameter in .env as we still need it to configure database timezone.

**Comment by julien4215** [comment]
I am converting this PR to draft until we decide if we keep supporting both postgresql and sqlite.

File: app/utils/types/datetime.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from sqlalchemy import DateTime
+from sqlalchemy.types import TypeDecorator
+
+
+class TZDateTime(TypeDecorator):
```
Comment (armanddidierjean) [inline app/utils/types/datetime.py:7]: Could you add comments explaining when are called these two functions and where is the class used?

**Review by armanddidierjean — APPROVED** [review]

**Review by guillaumetavernier — APPROVED** [review]

**Pull request merged**